src/pages/api/preview.js

```diff
@@ -0,0 +1,4 @@
+export default function handler(req, res) {
+    res.setPreviewData({});
+    res.redirect('/developer/preview' + req.query.redirect);
+}
```

Comment:
Are we ok with having unpublished articles publicly accessible through this preview route?

Reply:
I'm trying to add a secret key, which is needed to be passed for preview to work. That secret key must be sent from Strapi and it will be stored in our env / secret variables.